src/table.ts

```diff
@@ -592,25 +588,21 @@ class Table {
    * region_tag:spanner_read_data_with_storing_index
    * Reading data using a storing index:
    */
-  read(request: ReadRequest, options?: TimestampBounds): ReadPromise;
-  read(request: ReadRequest, callback: ReadCallback): void;
-  read(request: ReadRequest, options: TimestampBounds, callback: ReadCallback):
-      void;
   read(
-      request: ReadRequest, options?: TimestampBounds|ReadCallback,
-      callback?: ReadCallback): ReadPromise|void {
-    const rows: Array<Row|Json> = [];
-
-    if (is.fn(options)) {
-      callback = options as ReadCallback;
-      options = {} as TimestampBounds;
-    }
+      request: ReadRequest, optionsOrCallback?: TransactionOptions|ReadCallback,
```

Comment:
Pretty sure we want to use `TimestampBounds`, but if that is not the case then we need to update the casts below to use `TransactionOptions` instead.